### PR TITLE
Reduces omega weed reagent volume

### PR DESCRIPTION
**Pull request by YakumoChen**
Duuuuude....

## About The Pull Request

Removes the ludicrous volume of omega weed

## Why It's Good For The Game

It is simply too dank , when you consider what dumb chemistry shit you can do with it and kitchen stuff.

## Changelog
:cl: YakumoChen
tweak: Duuuude, space narcs took our ultravolume 420 omega weed. Not cool brah.
/:cl:

### CODE REVIEW

**Comment by nemvar** [comment]
Needs a changelog

**Comment by YakumoChen** [comment]
Done.

**Comment by CRITAWAKETS** [comment]
Maybe a bit too little volume with this change. Could maybe make it something like 133.7

**Comment by Fikou** [comment]
:grimacing:

File: code/modules/hydroponics/grown/cannabis.dm [hunk 1]
```diff
@@ -117,5 +117,5 @@
 	name = "omega cannabis leaf"
 	desc = "You feel dizzy looking at it. What the fuck?"
 	icon_state = "ocannabis"
-	volume = 420
+	volume = 42.0
```
Comment (81Denton) [inline code/modules/hydroponics/grown/cannabis.dm]: This is less than the default 50 volume and makes no sense for a T2 plant.

**Comment by Fikou** [comment]
this makes no sense anyway but 69 or 133.7 would be better

File: code/modules/hydroponics/grown/cannabis.dm (hunk 1 above)
Comment (YakumoChen) [reply code/modules/hydroponics/grown/cannabis.dm]: I wanted to preserve the joke but ok

**Comment by YakumoChen** [comment]
Theres already a 69 joke in cannabis.dm, not sure if it really needs another one.

**Comment by ExcessiveUseOfCobblestone** [comment]
can you provide some context? 

> when you consider what dumb chemistry shit you can do with it and kitchen stuff.

I have and not sure what you're referring to

**Comment by YakumoChen** [comment]
I was told you can add plants to custom burgers to add all the reagents and further continue the meme with frying. I would also assume (and was told) you can double the volume with the double volume plant trait, giving it 840 volume. You can make some scary huge reagent containers which could lead to, say, making a giant nitro bomb burger.

**Comment by YakumoChen** [comment]
Capture+_2019-10-20-19-20-51

**Comment by CRITAWAKETS** [comment]
I mean, considering the effort it takes to do, it would be better to nerf it instead of removing the mechanic outright.

**Comment by 81Denton** [comment]
Plant volume is the cap of how many chemicals fit in there - plant traits are always fixed amounts and don't increase with a larger volume.
They're wrongly shown as percentages ingame, but in reality you just have a 420 volume plant of which most ends up unused.

**Comment by YakumoChen** [comment]
You can inject more with a syringe though

**Comment by DaedalusGame** [comment]
i don't really know why you wouldn't A) cap burger volumes then or B) wait for the first time a 840u burger bomb is detonated on a live server

**Comment by YakumoChen** [comment]
You can put 10 of them in a burger. So itd be a 8400u bomb.
It was done to EMP the whole level on a downstream. I assume it could probably cause a crash or obliterate an entire round.

**Comment by YakumoChen** [comment]
I think in any case it isnt a good idea to have growable super bluespace beakers, in case someone finds a different and worse way to exploit it.

**Comment by ExcessiveUseOfCobblestone** [comment]
fyi this doesn't fix that issue, it just makes it require more loops

**Comment by YakumoChen** [comment]
Is that a reason not to merge this?

**Comment by Krysonism** [comment]
Will the omega weed chems even fit now?

**Comment by carlarctg** [comment]
cringe pr, no reason to remove fun

**Comment by 81Denton** [comment]
@YakumoChen Thanks for elaborating. IMO the better fix would be capping burger max volumes so you can't fit a few gallons worth of chemicals into a single burger.

**Comment by Shadowflame909** [comment]
Meme numbers must be replaced with meme numbers

for a slight nerf you could do a 360 no-scope

but I don't really use these for kill mixes anyways. Amatoxin limes always get the job done.

**Comment by Quartich** [comment]
Please don't kill botany more than it already is

**Comment by YakumoChen** [comment]
@81Denton thats fine, and should be done (probably by someone who can do more than tweak numbers), I'd still push for this to be merged. The volume only exists to be a meme and isn't valuable except to make bad exploits like this one worse.

**Comment by ATH1909** [comment]
Bro, now omega weed doesn't have enough room for all of its chems at 100 potency.

Also, you can't really use these as bluespace beakers very well, as they (obviously) can't fit in many machines (including grenades) and you can't use a syringe to extract chems from omega weed leaves (you can only inject (not extract) more chems into them).

**Comment by ATH1909** [comment]
> You can put 10 of them in a burger. So itd be a 8400u bomb.
> It was done to EMP the whole level on a downstream. I assume it could probably cause a crash or obliterate an entire round.

How did they manage to inject hundreds of units of chems simultaneously? Wouldn't you need to keep the chems from reacting until you finish injecting them all, which would require the separated chemicals trait (which we don't have) or the like?

**Comment by ATH1909** [comment]
Also, I just looked at the code for adding ingredients to customizable foods, and I can't find anything that'd increase the maximum chem capacity of the final product.

```
/obj/item/reagent_containers/food/snacks/customizable/attackby(obj/item/I, mob/user, params)
	if(!istype(I, /obj/item/reagent_containers/food/snacks/customizable) && istype(I, /obj/item/reagent_containers/food/snacks))
		var/obj/item/reagent_containers/food/snacks/S = I
		if(I.w_class > WEIGHT_CLASS_SMALL)
			to_chat(user, "<span class='warning'>The ingredient is too big for [src]!</span>")
		else if((ingredients.len >= ingMax) || (reagents.total_volume >= volume))
			to_chat(user, "<span class='warning'>You can't add more ingredients to [src]!</span>")
		else if(istype(I, /obj/item/reagent_containers/food/snacks/pizzaslice/custom) || istype(I, /obj/item/reagent_containers/food/snacks/cakeslice/custom))
			to_chat(user, "<span class='warning'>Adding [I.name] to [src] would make a mess.</span>")
		else
			if(!user.transferItemToLoc(I, src))
				return
			if(S.trash)
				S.generate_trash(get_turf(user))
			ingredients += S
			mix_filling_color(S)
			S.reagents.trans_to(src,min(S.reagents.total_volume, 15), transfered_by = user) //limit of 15, we don't want our custom food to be completely filled by just one ingredient with large reagent volume.
			foodtype |= S.foodtype
			update_snack_overlays(S)
			to_chat(user, "<span class='notice'>You add the [I.name] to the [name].</span>")
			update_name(S)
	else
		. = ..()
```

You said that this stunt was done on a downstream, right? Do you know which downstream it was?

**Comment by 81Denton** [comment]
If there are still issues after fixing the borgar bug we can look at this again.

**Comment by ExcessiveUseOfCobblestone** [comment]
> 
> 
> Is that a reason not to merge this?

yes when that is your reason for adding it...

**Pull request closed without merging**